src/graphics/shader/shadermaterial.h

```diff
@@ -38,7 +38,7 @@ class ShaderMaterial {
 		MATERIAL_TRANSPARENT   = 0x0002,  ///< Material definitely has transparency.
 		MATERIAL_TRANSPARENT_B = 0x0004,  ///< Material definitely has transparency.
 		MATERIAL_DECAL         = 0x0008,  ///< Material is a decal.
-		MATERIAL_SPECIAL_BLEND = 0x0010   ///< Different to default blending. Maybe this should be extended one day.
+		MATERIAL_CUSTOM_BLEND  = 0x0010,  ///< Different blending is being used, see _blendMode.
```

Comment:
Argh, note to self: bad comment on variable name. Will clean up after after comments are added.